.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    trailingComma: 'none',
+    arrowParens: 'avoid',
+    // parser: "typescript",
```

Comment:
why it's commented? Either un comment it or remove the line

Reply:
In my opinion the line should be removed to rely on parser auto-detection, as clearly stated in prettier doc.
https://prettier.io/docs/en/configuration.html , section "Setting the parser option"

Reply:
yes, I agree with you, so we should completely remove the line (not just comment it) - in general we try to avoid committing commented code - if you really need to commit commented code you should add an additional comment explaining why there's some commented out code there :)

Reply:
Sounds like recursive hell :-)